pkg/rbd/nodeserver.go

```diff
@@ -218,6 +218,10 @@ func (ns *nodeServer) NodeUnstageVolume(
 	return nil, status.Error(codes.Unimplemented, "")
 }
 
+func (ns *nodeServer) NodeGetInfo(ctx context.Context, req *csi.NodeGetInfoRequest) (*csi.NodeGetInfoResponse, error) {
```

Comment:
do we need this one? I was in an assumption that if we don't implement this method it will call the NodeGetInfo on default node server.

Reply:
this is called by node-registrar https://github.com/kubernetes-csi/node-driver-registrar/blob/master/pkg/connection/connection.go#L122